models/project.py

```diff
@@ -7,6 +7,6 @@ class Tasks(models.Model):
     _name = "project.task"
     _inherit = "project.task"
 
-    sale_order = fields.Many2one("sale.order")
+    sale_order = fields.Many2one("sale.order")  # This field is already defined in gse_purchase_tracking?
```

Comment:
To remove then?

Reply:
Yep, done, had a deeper look and it was not used anyway, the xml file using it was commented too.